src/NServiceBus.AcceptanceTests/Receiving/When_subscribed_to_OnOnMessageReceiveCompleted.cs

```diff
@@ -3,49 +3,50 @@
     using System;
     using System.Threading.Tasks;
     using AcceptanceTesting;
-    using Features;
     using EndpointTemplates;
-    using NUnit.Framework;
+    using Features;
     using NServiceBus.Transport;
+    using NUnit.Framework;
 
-    class When_subscribed_to_OnOnMessageReceiveCompleted : NServiceBusAcceptanceTest
+    class When_subscribed_to_OnReceiveCompleted : NServiceBusAcceptanceTest
     {
         [Test]
-        public async Task Should_receive_notifications_for_successfull()
+        public async Task Should_receive_when_OnMessage_succeeds()
```

Comment:
I'd prefer to call it `Should_receive_event_when_OnMessage_succeeds` but I got worried about test name length.